admin/packaging.py

```diff
@@ -662,6 +662,16 @@ def __str__(self):
 
         # Cryptography hazmat bindings
         'package-installs-python-pycache-dir opt/flocker/lib/python2.7/site-packages/cryptography/hazmat/bindings/__pycache__/',  # noqa
+
```

Comment:
I had to add those errors to the ignored errors list. 
I know that the install-ieee-data belongs here, but I am not that sure about
+        "package-contains-timestamped-gzip",
+        "systemd-service-file-outside-lib",
The first one seems like a reasonable things to add to the ignore list, but I know they were adding systemd in this new 15.10 release, so we may need some extra refactoring to use systemd instead of service?